lib/kb_Metrics/metricsDBs.py

```diff
@@ -87,133 +80,116 @@ def update_activity_records(self, upd_filter, upd_data):
             # return an instance of UpdateResult(raw_result, acknowledged)
             update_ret = self.mt_coll.update_one(upd_filter,
                                                  upd_op, upsert=True)
-        except BulkWriteError as bwe:
-            # pprint(bwe.details['writeErrors'])
-            panic = filter(lambda x: x['code'] != 11000,
-                           bwe.details['writeErrors'])
-            if len(panic) > 0:
-                print "really panic"
-                raise
-        else:
-            pass
-            # pprint(update_ret.raw_result)
-            # if update_ret.upserted_id:
-            # print(update_ret.upserted_id)
+        except WriteError as we:
+            pprint(we.details['writeErrors'])
```

Comment:
undefined pprint.  I strongly suggest you to test this.

Reply:
I will fix it.